src/Apps/Collect2/Components/Collection/Header/__tests__/Header.test.tsx

```diff
@@ -321,7 +324,7 @@ describe("collections header", () => {
 
       expect(entityHeaders.length).toEqual(4) // 4 = 3 artists + 1 "View More"
 
-      const viewMore = component.find(ViewMore)
+      const viewMore = component.find("ViewMore")
```

Comment:
Out of curiosity, is this a preference thing, or is there an objective argument to `component.find` by string instead of component reference?

Reply:
Less complexity IMO -- with an exported component, one 
a) has now made something that's effectively private, public
b) added an additional import

Most React components automatically have `displayName` attached, so its seamless to piggy back on that. Often, though, one needs to operate on a component (like `find(Foo).instance().callSomeMethod()` which will error out in TS if using string access. But this case is less common.